README.md

```diff
@@ -23,6 +23,18 @@ Quick Start Guide
 The server(s) can then be stopped with `python quickstart.py --stop`. This will
 only work if quickstart was used to start the server(s).
 
+Quick Start Guide (with docker)
+-------------------------------
+
+If you have Docker installed and working, you can do:
+
+1. `docker-compose build`
+2. `docker-compose up`
+
+Then use the sym server at http://localhost:8080 .
+
+You can stop the docker container by hitting `CTRL-C` twice.
+
```

Comment:
I didn't really know how you wanted to document the docker bits or whether your users typically have make installed or what. So these could be `make build` and `make run`, too.